app/src/main/res/layout/activity_scrolling.xml

```diff
@@ -24,6 +24,17 @@
             app:layout_constraintTop_toTopOf="parent"
             app:popupTheme="@style/ThemeOverlay.AppCompat.Light" />
 
+
+        <EditText
+            android:id="@+id/search_bar_Scrolling_EditText"
```

Comment:
minor correction: `editText` instead of `EditText`

Reply:
Thanks!